version 0.2

jible opened the pull request:
- World generation now generates structures 
- Stuctures are prefabs built on tilemaps, so they can be placed into the world tilemap seamlessly
- See chest room structure as example
- Created structure details resource

jible committed:
successfully added one structure

jible committed:
can create multiple structures

fixed render bug
debug print no longer mirror (works correctly)

jible committed:
changelog

jmclane21 reviewed (approved):
Beautiful fast code that we will clean :D

sym-z reviewed (approved):
Looks great! Excited to clean this up a bit.

jmclane21 committed:
spacing changes and cleanup

changed function name of check_for_type to distinguish purpose from get_tile_type
